mw-webapp/src/component/sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,41 @@
+import {ReactElement, useState} from "react";
```

Comment:
Files Sidebar, SidebarTrigger and SidebarContent looks very similar to Modal component. Is it right? If yes, can't we use modal component for render sidebar afor don't copy components? Please check)